src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -18,41 +18,32 @@ package com.twcable.grabbit.server.batch.steps.jcrnodes
 
 import com.google.protobuf.ByteString
 import com.twcable.grabbit.DateUtil
-import com.twcable.grabbit.proto.NodeProtos
-import com.twcable.grabbit.proto.NodeProtos.Node
-import com.twcable.grabbit.proto.NodeProtos.Properties
-import com.twcable.grabbit.proto.NodeProtos.Property
+import com.twcable.grabbit.proto.NodeProtos.Node as ProtoNode
```

Comment:
I've seen these red rectangles from time to time; but I can't seem to find any information about them. Anyone know what these are?